modules/openapi-generator/src/main/java/org/openapitools/codegen/CodegenConfig.java

```diff
@@ -100,6 +100,8 @@ public interface CodegenConfig {
 
     String escapeUnsafeCharacters(String input);
 
+    String escapeUnsafeCharactersNullGuard(String input);
```

Comment:
I chose this name as that is what it currently does. Most implementations of `escapeUnsafeCharacters` are not actually about unsafe characters but about multiline comments, so we'd have the chance to name this `escapeMultilineComment` if we wanted, which would be more obvious, but has the caveat that the function it calls out to (`escapeUnsafeCharacters`) might not be aligned, so I left it with this slightly awkward suffix.